rocketpy/plots/rocket_plots.py

```diff
@@ -86,27 +86,31 @@ def stability_margin(self):
 
         return None
 
-    def power_on_drag(self):
-        """Plots power on drag of the rocket as a function of time.
-
-        Returns
-        -------
-        None
-        """
-
-        self.rocket.power_on_drag()
-
-        return None
-
-    def power_off_drag(self):
-        """Plots power off drag of the rocket as a function of time.
+    def power_on_and_off_drag(self):
+        """Plots power off and on drag curve of the rocket as a function of time.
 
         Returns
         -------
         None
         """
+        x_power_drag_off = self.rocket.power_off_drag.x_array
+        y_power_drag_off = self.rocket.power_off_drag.y_array
+        x_power_drag_on = self.rocket.power_on_drag.x_array
+        y_power_drag_on = self.rocket.power_on_drag.y_array
+
+        fig, ax = plt.subplots()
+        ax.plot(x_power_drag_on, y_power_drag_on, label="Power on Drag")
+        ax.plot(
+            x_power_drag_off, y_power_drag_off, label="Power off Drag", linestyle="--"
+        )
 
-        self.rocket.power_off_drag()
+        ax.set_title("Power on and off Drag")
```

Comment:
A small suggestion regarding the title (feel free to refuse it): "Drag curves".

Reply:
I agree, it will look better